obspy/io/mseed/scripts/recordanalyzer.py

```diff
@@ -262,6 +263,114 @@ def _parse_blockette(self, blkt_type):
                     raise IOError(msg)
                 raise
             blkt_dict['Sampling Rate'] = float(unpack_values[0])
+        elif blkt_type == 300:
+            # Step calibration blockette
+            _tmp = self.file.read(56)
+            try:
+                unpack_values = unpack(native_str('%sHHBBBBHBBLLf3sxL12s12s'
+                                                  % self.endian), _tmp)
+            except Exception:
+                if len(_tmp) == 0:
+                    msg = "Unexpected end of file."
+                    raise IOError(msg)
+                raise
+            blkt_dict['Calibration Start Time'] = \
+                UTCDateTime(year=int(unpack_values[0]),
+                            julday=int(unpack_values[1]),
+                            hour=int(unpack_values[2]),
+                            minute=int(unpack_values[3]),
+                            second=int(unpack_values[4]),
+                            microsecond=int(unpack_values[5]))
+
+            blkt_dict['Number of Step Calibrations'] = int(unpack_values[7])
+            blkt_dict['Step Duration'] = int(float(unpack_values[8])/10000.)
```

Comment:
maybe call this "Step Duration in Seconds"? Just so its not confusing for people who use this with the reference manual at hand.